.travis.yml

```diff
@@ -1,18 +1,17 @@
 language: node_js
 
 node_js:
-  - 17
+  - 16
 
 cache:
-  # cache both npm modules and Cypress binary
+  # cache both npm modules
   directories:
     - ~/.npm
     - ~/.cache
-  # install dependencies and check that Cypress can run
-  override:
-    - npm ci
-    - npm run cy:verify
 
 script:
+  - npm run lint
+  - npm run check
+  - npx playwright install
```

Comment:
are you sure `npx playwright install` should be in scripts section, or `before_install` or something like this? please investigate it